yamls/baselines/bert-base-uncased.yaml

```diff
@@ -15,23 +12,19 @@ tokenizer_name: bert-base-uncased
 # Base model config
 model:
   name: hf_bert
```

Comment:
This threw me off before I remembered this loads `AutoModel` rather than `BertModel` -- I think we should rename that to `hf_model` or something 😅